docs/content/kafka-authorization.md

```diff
@@ -25,87 +25,115 @@ rules to reuse logic and improve overall readability.
+    image: confluentinc/cp-zookeeper:6.2.1
+    ports:
+      - "2181:2181"
     environment:
-      ZOOKEEPER_CLIENT_PORT: 2181
-      zk_id: "1"
-  kafka:
-    hostname: kafka
-    image: openpolicyagent/demo-kafka:1.0
-    links:
-      - zookeeper
-      - opa
+      - ALLOW_ANONYMOUS_LOGIN=yes
+      - ZOOKEEPER_CLIENT_PORT=2181
+  broker:
+    image: confluentinc/cp-kafka:6.2.1
     ports:
-      - "9092:9092"
+      - "9093:9093"
     environment:
-      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: "1"
+      # Set cache expiry to low value for development in order to see decisions
+      KAFKA_OPA_AUTHORIZER_CACHE_EXPIRE_AFTER_SECONDS: 10
+      KAFKA_OPA_AUTHORIZER_URL: http://opa:8181/v1/data/kafka/authz/allow
+      KAFKA_AUTHORIZER_CLASS_NAME: org.openpolicyagent.kafka.OpaAuthorizer
+      KAFKA_BROKER_ID: 1
       KAFKA_ZOOKEEPER_CONNECT: "zookeeper:2181"
-      KAFKA_ADVERTISED_LISTENERS: "SSL://:9093"
+      KAFKA_ADVERTISED_LISTENERS: SSL://localhost:9093
       KAFKA_SECURITY_INTER_BROKER_PROTOCOL: SSL
+      KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR: 1
+      KAFKA_GROUP_INITIAL_REBALANCE_DELAY_MS: 0
+      KAFKA_TRANSACTION_STATE_LOG_MIN_ISR: 1
+      KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR: 1
+      KAFKA_AUTO_CREATE_TOPICS_ENABLE: "true"
+      KAFKA_SSL_KEYSTORE_FILENAME: server.keystore
+      KAFKA_SSL_KEYSTORE_CREDENTIALS: credentials.txt
+      KAFKA_SSL_KEY_CREDENTIALS: credentials.txt
+      KAFKA_SSL_TRUSTSTORE_FILENAME: server.truststore
+      KAFKA_SSL_TRUSTSTORE_CREDENTIALS: credentials.txt
       KAFKA_SSL_CLIENT_AUTH: required
-      KAFKA_SSL_KEYSTORE_FILENAME: kafka.broker.keystore.jks
-      KAFKA_SSL_KEYSTORE_CREDENTIALS: broker_keystore_creds
-      KAFKA_SSL_KEY_CREDENTIALS: broker_sslkey_creds
-      KAFKA_SSL_TRUSTSTORE_FILENAME: kafka.broker.truststore.jks
-      KAFKA_SSL_TRUSTSTORE_CREDENTIALS: broker_truststore_creds
-      KAFKA_AUTHORIZER_CLASS_NAME: com.lbg.kafka.opa.OpaAuthorizer
-      KAFKA_OPA_AUTHORIZER_URL: "http://opa:8181/v1/data/kafka/authz/allow"
-      KAFKA_OPA_AUTHORIZER_ALLOW_ON_ERROR: "false"
-      KAFKA_OPA_AUTHORIZER_CACHE_INITIAL_CAPACITY: 100
-      KAFKA_OPA_AUTHORIZER_CACHE_MAXIMUM_SIZE: 100
-      KAFKA_OPA_AUTHORIZER_CACHE_EXPIRE_AFTER_MS: 600000
-```
-
-For more information on how to configure the OPA plugin for Kafka, see the [github.com/open-policy-agent/contrib](https://github.com/open-policy-agent/contrib)
-repository.
-
-Once you have created the file, launch the containers for this tutorial.
-
-```bash
-docker-compose --project-name opa-kafka-tutorial up
+      CLASSPATH: "/plugin/*"
+    volumes:
+      - "./plugin:/plugin"
+      - "./cert/server:/etc/kafka/secrets"
+    depends_on:
+      - opa
+      - zookeeper
 ```
 
-Now that the tutorial environment is running, we can define an authorization policy using OPA and test it.
-
 #### Authentication
 
 The Docker Compose file defined above requires **SSL client authentication**
```

Comment:
I slightly prefer SSL -> TLS, but it's OK to keep as-is.

Reply:
Agreed, and even the Kafka docs complain about their own terminology, but the config options are all like:

```
      KAFKA_SSL_KEYSTORE_FILENAME: server.keystore
      KAFKA_SSL_KEYSTORE_CREDENTIALS: credentials.txt
      KAFKA_SSL_KEY_CREDENTIALS: credentials.txt
      KAFKA_SSL_TRUSTSTORE_FILENAME: server.truststore
      KAFKA_SSL_TRUSTSTORE_CREDENTIALS: credentials.txt
      KAFKA_SSL_CLIENT_AUTH: required
```
So in light of that, I think it's OK to stick with what they have established in this context.